core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -354,10 +350,14 @@ public JDBCUserStoreManager(RealmConfiguration realmConfig, Map<String, Object>
 			try {
 				rs = prepStmt.executeQuery();
 			} catch (SQLException e) {
-				// may be due time out, therefore ignore this
-				log.error(e);
-				return users;
-			}
+                if(e instanceof SQLTimeoutException){
+                    log.error("may be due time out, therefore ignore this",e);
```

Comment:
"The cause might be a time out. Hence ignored." - Just trying to format the message